tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -2931,6 +2931,21 @@ static function() {
 		$this->assertSame( $expected, $print_scripts );
 	}
 
+	/**
+	 * Ensure tinymce scripts aren't loading async.
+	 *
+	 * @ticket 58648
+	 */
+	public function test_printing_tinymce_scripts() {
```

Comment:
Not sure if it's covered elsewhere but a similar test for scripts registered with a loading strategy but never have `wp_enqueue_script()` called might be helpful.

_might_ being the keyword. This isn't a passive agressive change request -- could be added later so this can be committed pre beta.

Reply:
Good suggestion. I've added one in [d3835ba](https://github.com/WordPress/wordpress-develop/pull/4852/commits/d3835ba958520fd2c99032b13ddd7e600da938cf).